lib/board.rb

```diff
@@ -0,0 +1,52 @@
+class Board
+	attr_accessor :cells, :turns_taken
+	def initialize
+		@cells = Array.new(9, " ")
+		@turns_taken = []
+	end
+
+	def reset!
+		@cells = Array.new(9, " ")
+	end
+
+	def display
+		puts " #{cells[0]} | #{cells[1]} | #{cells[2]} ", 
+		"-----------", 
+		" #{cells[3]} | #{cells[4]} | #{cells[5]} ", 
+		"-----------", 
+		" #{cells[6]} | #{cells[7]} | #{cells[8]} "
+	end
+
+	def position(input)
+		return self.cells[input.to_i - 1]
+	end
+
+	def full?
+		self.cells.include?(" ") ? false : true
```

Comment:
Avoid using the ternary operator to return `true` and `false`. Instead, rely on the implicit boolean return value of the expression. You can reverse (or negate) the boolean with the `!` (bang) operator.
```
def full?
  !self.cells.include?(" ")
end
```